src/python/WMCore/WMBS/CreateWMBSBase.py

```diff
@@ -278,15 +278,15 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE TABLE wmbs_job (
              id           INTEGER       PRIMARY KEY AUTO_INCREMENT,
              jobgroup     INTEGER       NOT NULL,
-             name         VARCHAR(255),
+             name         VARCHAR(70),
              state        INTEGER       NOT NULL,
              state_time   INTEGER       NOT NULL,
              retry_count  INTEGER       DEFAULT 0,
              couch_record VARCHAR(255),
              location     INTEGER,
              outcome      INTEGER       DEFAULT 0,
-             cache_dir    VARCHAR(767)  DEFAULT 'None',
-             fwjr_path    VARCHAR(767),
+             cache_dir    VARCHAR(70)  DEFAULT 'None',
+             fwjr_path    VARCHAR(70),
```

Comment:
Valentin, to avoid forgetting that we do need to update these columns back to their original values before merging this PR, can you please update them now? It will probably be merged eventually, so better update the code now then let it go to production. Thanks